javascript/javascript-7-oop.md

```diff
@@ -21,6 +21,9 @@
 - [JavaScript 6 - Object Literals Lesson](/javascript/javascript-6-object-literals.md)
 
 ### Motivation
+- It helps to organize larger projects.
+- It support OOP because it supports inheritance through prototyping as well as properties and methods.
+- Here is the tutorial for object-oriented javascript for beginners[https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Objects/Object-oriented_JS]
```

Comment:
You had to create a section under motivation named `Why learn OOP in JavaScript?` as said in the issue. And have to add points under that section.

Reply:
I updated javascript-7-oop.md. Please verify now.